docs/relay/mutations.rst

```diff
@@ -12,13 +12,12 @@ subclass of ``relay.ClientIDMutation``.
 .. code:: python
 
     class IntroduceShip(relay.ClientIDMutation):
-
         class Input:
-            ship_name = graphene.String(required=True)
```

Comment:
I think I like this better without `graphene graphene graphene` everywhere!

Did we have a discussion with folks to confirm that everyone is generally on the same page? The current convention is followed in all kinds of places so it will take some doing to change all the code examples.

Reply:
I'm all for this. It makes the code much easier to read.

Reply:
@dvndrsn We had a brief discussion on Slack about this. One reason for this PR as WIP is to further the conversation. See: https://graphenetools.slack.com/archives/CGR4VCHUL/p1559524058000700

@jkimbo I feel the same way.